### ISSUE TITLE
Colors per dictionary

### ISSUE THREAD

**Issue by Snaptags** [itkach/aard2-android#53]
I think having the possibility to use a color per slob could increase visibility when using many different dictionaries at the same time:

![colors](https://cloud.githubusercontent.com/assets/1249745/13704281/5dd019ee-e79b-11e5-9acb-32bbb1691c9b.png)

I already tried adding <font color…> tags to the keys when converting the dict files to slob, but they end up as text in the slob. Besides this would really blow up the overall file size, since one color per slob file should be enough :-)

Could be an option during slob creation, or could be added to the dictionary manager tab in the aard2 gui…

**Comment by MHBraun** [comment]
Adding the colour in creating the dictionary will be inflexible for the users. They have different devices with different colour settings. 
It would be better to assign the colours within the Aard2 GUI



Von meinem Samsung Galaxy Smartphone gesendet.


-------- Ursprüngliche Nachricht --------
Von: Snaptags <notifications@github.com> 
Datum:11.03.2016  08:16  (GMT-06:00) 
An: itkach/aard2-android <aard2-android@noreply.github.com> 
Cc:  
Betreff: [aard2-android] Colors per dictionary (#53) 

I think having the possibility to use a color per slob could increase visibility when using many different dictionaries at the same time:



I already tried adding tags to the keys when converting the dict files to slob, but they end up as text in the slob. Besides this would really blow up the overall file size, since one color per slob file should be enough :-)

Could be an option during slob creation, or could be added to the dictionary manager tab in the aard2 gui…

**Comment by itkach** [comment]
> It would be better to assign the colours within the Aard2 GUI

I agree. Adding ability to assign color to dictionary in dictionary list doesn't sound too complicated.
